packages/components/src/Typeahead/Typeahead.component.renderers.js

```diff
@@ -95,17 +92,26 @@ export function renderItemsContainerFactory(
 		} else {
 			content = children;
 		}
-
 		return (
-			<div id={id} ref={ref} className={containerClassName} {...restProps}>
-				{render(content, { searching, loading, noResult, isShown })}
+			<div
+				className={containerClassName}
+				id={containerProps.id}
+				key={containerProps.key}
+				ref={containerProps.ref}
+				role={containerProps.role}
+			>
+				{render(content, {
+					searching,
+					loading,
+					noResult,
+					isShown,
+					refContainer: containerProps.ref,
```

Comment:
You pass the same ref for the div above and the render prop. Isn't that problematic ?